pkg/agent/agent.go

```diff
@@ -111,10 +128,73 @@ func (a *Agent) Run(healthCheck *metrics.HealthCheck, enableProfiling bool, bind
 	}()
 
 	<-a.ctx.Done()
+
+	_ = a.DeleteNodeResourceTsp()
 }
 
 func getAgentName(nodeName string) string {
-	return nodeName + "_" + string(uuid.NewUUID())
+	return nodeName + "." + string(uuid.NewUUID())
+}
+
+func (a *Agent) CreateNodeResourceTsp() string {
+	tsp, err := a.craneClient.PredictionV1alpha1().TimeSeriesPredictions(resource.TspNamespace).Get(context.TODO(), a.GenerateNodeResourceTspName(), metav1.GetOptions{})
```

Comment:
As the tsp is deleted and then created, the predict module will  recalculate ？ have long time is to be waited?

Reply:
The forecast module is immediately recalculated. The purpose of my deletion was to ensure that the tsp used by the crane-agent was created by itself, to ensure that it would not duplicate the tsp created by other modules, so I used the name of the agent to name the tsp, but the name of the agent was regenerated every time the agent started, so I had to recreate the tsp.